File: app/src/main/java/se/curtrune/lucy/web/VersionInfo.java
package se.curtrune.lucy.web;

public class VersionInfo {
    private String versionName;
    private String fileName;
    private int versionCode;
    private String versionInfo;
    private String url;


    public String getFileName(){
        return fileName;
    }
    public String getUrl(){
        return url;
    }
    public String getVersionInfo(){
        return versionInfo;
    }

    public String getVersionName() {
        return versionName;
    }

    public int getVersionCode() {
        return versionCode;
    }

    @Override
    public String toString() {
        return "VersionInfo{" +
                "versionName='" + versionName + '\'' +
                ", versionCode=" + versionCode +
                '}';
    }
}

File: app/src/main/java/se/curtrune/lucy/web/VersionInfo.kt
package se.curtrune.lucy.web

import kotlinx.serialization.Serializable

@Serializable
data class VersionInfo(
    @JvmField
    val versionName: String? = null,
    @JvmField
    val versionCode: Int = 0,
    @JvmField
    val versionInfo: String? = null,
    @JvmField
    val url: String? = null
)
